setup.py

```diff
@@ -230,6 +230,10 @@ def prepare_env_for_osx():
         description=
         """Python API for cpptraj: a data analysis package for biomolecular simulation""",
         license="GPL v3",
+        install_requires=['numpy', 'libmagic'],
+        tests_require=['tqdm', 'traitlets', 'IPython', 'mpi4py', 'pytest', 'cclib',
+                       'pysander', 'ipywidgets', 'nglview', 'ipykernel'],
```

Comment:
exclude `pysander`, it does not have pip package.

Reply:
Ok. Is https://pypi.org/project/pysander/ something else?

Reply:
... it seems it is.

While it's a bit dangerous to add a dependency on a similarly named public package on PyPI that is not the one you want, one way to solve this is to require a specific version that is not available on https://pypi.org/project/pysander/#files, e.g.: `pysander>=16.0` if that is the version of pysander that is installed by other means.

Then pip install will complain if the user did not preinstall `pysander` by other means before running tests.

Reply:
the `pysander` used by `pytraj` comes from `AmberTools`.

Reply:
Ok. I've removed the dependency and let it be up to you if you want to keep it, with a version restriction to make sure we pick up the AmberTools version, or leave it removed.

Reply:
Another solution, of course, could be to upload the AmberTools version of `sander` to PyPI under a different name, e.g. `ambertools-sander`, to have it available for pip installation.

Reply:
> Another solution, of course, could be to upload the AmberTools version of sander to PyPI under a different name, e.g. ambertools-sander, to have it available for pip installation.

it's to complicated process (in case you're not familiar with AmberTools) with no real benefit. So just let's remove its requirement. We need to balance between usability and maintenance effort.